kubeadm/scripts/PrepareNode.ps1

```diff
@@ -136,4 +142,4 @@ if ($ContainerRuntime -eq "Docker") {
     nssm set kubelet DependOnService containerd
 }
 
-New-NetFirewallRule -Name kubelet -DisplayName 'kubelet' -Enabled True -Direction Inbound -Protocol TCP -Action Allow -LocalPort 10250
+New-NetFirewallRule -Name kubelet -DisplayName 'kubelet' -Enabled True -Direction Inbound -Protocol TCP -Action Allow -LocalPort 10250
```

Comment:
unnecessary change, plus there's an error here: "no newline at the end of file".